lib/streamlit/DeltaGenerator.py

```diff
@@ -2523,7 +2524,7 @@ def progress(self, element, value):
                 )
         else:
             raise StreamlitAPIException(
-                "Progress Value has invalid type: %s" % value_type
+                "Progress Value has invalid type: %s" % type(value)
```

Comment:
This should have `.__name__` at the end like before so we see, e.g. `str` rather than `<class 'str'>`.

Reply:
Sure, ok.